mitodl/mit-open
# topic / privacy / onboarding / profile copy updates

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4816

### Description (What does it do?)
This PR makes a number of different copy edits throughout the app, as described in the issue. After updating the headings for top level topic pages, I noticed an issue with the display of said heading. It was being displayed twice; once below the banner and once inside the banner. I removed the display from below the banner, as it wasn't supposed to be there.

### How can this be tested?
 - Make sure you have `RUN_DATA_MIGRATIONS=True` in your environment
 - Spin up `mit-open` on this branch
 - Go over the requested changes in the above issue and make sure they have been implemented:
   - Topic listing: http://localhost:8063/topics/
   - Top level topic channel pages: click on any of the top level topics and ensure that they have a heading matching the one in the issue
   - Department listing: http://localhost:8063/topics/
   - Onboarding: http://localhost:8063/onboarding
   - Profile: http://localhost:8063/dashboard/#profile
   - Privacy Policy: http://localhost:8063/privacy
   - Homepage "personalize" section: http://localhost:8063/
